### PR TITLE
rocrate: Update crate with version bump and handle new contributor field

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

File: nf_core/pipelines/rocrate.py [hunk 1]
```diff
@@ -336,6 +348,28 @@ def add_main_authors(self, wf_file: rocrate.model.entity.Entity) -> None:
...
+        """
+        # check if we need to output a json file and/or a zip file based on the file extensions
+        json_path = None
+        zip_path = None
+        # try to find a json file
+        json_path = Path(self.pipeline_dir, "ro-crate-metadata.json")
+        if json_path.exists():
+            json_path = json_path
+        else:
+            json_path = None
```
Comment (mirpedrol) [inline nf_core/pipelines/rocrate.py:363]: Not needed
```suggestion
```

File: nf_core/pipelines/rocrate.py [hunk 2]
```diff
@@ -336,6 +348,28 @@ def add_main_authors(self, wf_file: rocrate.model.entity.Entity) -> None:
...
+
+        # try to find a zip file
+        zip_path = Path(self.pipeline_dir, "ro-crate.crate.zip")
+        if zip_path.exists():
+            zip_path = zip_path
+        else:
+            zip_path = None
```
Comment (mirpedrol) [inline nf_core/pipelines/rocrate.py:370]:
```suggestion
```

**Review by mirpedrol — APPROVED** [review]

File: nf_core/pipelines/rocrate.py [hunk 3]
```diff
@@ -336,6 +348,28 @@ def add_main_authors(self, wf_file: rocrate.model.entity.Entity) -> None:
             if author in authors:
                 wf_file.append_to("maintainer", author_entitity)
 
+    def update_rocrate(self) -> bool:
+        """
+        Update the rocrate file
+        """
+        # check if we need to output a json file and/or a zip file based on the file extensions
+        json_path = None
+        zip_path = None
```
Comment (mashehu) [inline nf_core/pipelines/rocrate.py:357]:
```suggestion
        json_path = None
        zip_path = None
```
these are actually not needed

**Comment by mashehu** [comment]
@nf-core-bot fix linting

**Pull request merged**